### PR TITLE
813 task add test for progress with labels

### PR DESCRIPTION
**Pull request by AlexandraKozhevnikova**

### CODE REVIEW

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/ProgressWithLabelsTests.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    }
+
+    @Test
+    public void getDefaultPercentTest() {
+        assertThat(progressWithLabels.core().getText(), is(defaultPercent));
+    }
+
+    @Test
+    public void getPercentTest() {
+        assertThat(progressWithLabels.core().getText(), is(defaultPercent));
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/ProgressWithLabelsTests.java]: why don't  we want to use something like progressWithLabels.core().is().text(defaultPercent) here?

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/Progress.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.epam.jdi.light.ui.bootstrap.elements.common;
+
+import com.epam.jdi.light.asserts.generic.UIAssert;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+
+public class Progress extends UIBaseElement<UIAssert> {
+
```
Comment (mf-code) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/Progress.java]: extra empty strings

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/ProgressWithLabelsTests.java (hunk 1 above)
Comment (AlexandraKozhevnikova) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/ProgressWithLabelsTests.java]: Thank you)) 
I didn't find this way. I tried without ".core" (((

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/Progress.java (hunk 2 above)
Comment (AlexandraKozhevnikova) [reply jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/Progress.java]: done)

**Pull request merged**